test/API/tattr.c

```diff
@@ -5911,31 +5901,31 @@ attr_info_by_idx_check(hid_t obj_id, const char *attrname, hsize_t n, hbool_t us
     if (HDstrcmp(attrname, tmpname) != 0)
         TestErrPrintf("Line %d: attribute name size wrong!\n", __LINE__);
 
-        /* Don't test "native" order queries on link name order, since there's not
-         *  a good way to easily predict the order of the links in the name index.
-         */
+    /* Don't test "native" order queries on link name order, since there's not
+     *  a good way to easily predict the order of the links in the name index.
+     */
 
-#ifndef NO_DECREASING_ALPHA_ITER_ORDER
-    /* Verify the information for first attribute, in decreasing name order */
-    HDmemset(&ainfo, 0, sizeof(ainfo));
-    ret = H5Aget_info_by_idx(obj_id, ".", H5_INDEX_NAME, H5_ITER_DEC, n, &ainfo, H5P_DEFAULT);
-    CHECK(ret, FAIL, "H5Aget_info_by_idx");
-    VERIFY(ainfo.corder, 0, "H5Aget_info_by_idx");
+    if (vol_cap_flags_g & H5VL_CAP_FLAG_CREATION_ORDER) {
```

Comment:
This block actually doesn't need a check for creation order support because these calls are using H5_INDEX_NAME to search alphabetically instead of by object creation order.

Reply:
Should I remove the check entirely, then?

Reply:
In this yes I'd just go ahead and remove the check. It sort of fits under H5VL_CAP_FLAG_ITERATE, but not exactly so for now we can just leave out any checking and revisit it later.

Reply:
Okay, it's been removed.